microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -105,20 +143,32 @@ The item ID is a string that represents the primary key of the items.  In the Mi
 
 The item's `price` is a `PaymentCurrencyAmount` that contains the current price of the item in the user's current region and currency.  The `price` is designed to be formatted for the user's current locale by using `Intl.NumberFormat`, as shown above.
 
-See also:
-* [StoreProduct.InAppOfferToken Property](/uwp/api/windows.services.store.storeproduct.inappoffertoken)
-* [PaymentCurrencyAmount dictionary](https://www.w3.org/TR/payment-request/#dom-paymentcurrencyamount) in _Payment Request API_ at W3C.
+
+<!-- ------------------------------ -->
+#### See also
+
+Digital Goods API: Draft Community Group Report
+* [DigitalGoodsService interface](https://wicg.github.io/digital-goods/#digitalgoodsservice) in _Digital Goods API: Draft Community Group Report_.
+   * [getDetails() method](https://wicg.github.io/digital-goods/#getDetails-method) in _Digital Goods API: Draft Community Group Report_.
+
+Learn.microsoft.com:
+* [StoreProduct.InAppOfferToken Property](/uwp/api/windows.services.store.storeproduct.inappoffertoken).
+
+W3C:
+* [PaymentCurrencyAmount dictionary](https://www.w3.org/TR/payment-request/#dom-paymentcurrencyamount) in _Payment Request API: W3C Candidate Recommendation Draft_.
+
+MDN:
 * [Intl.NumberFormat](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) at MDN.
```

Comment:
```suggestion
Digital Goods API specification:
* [getDetails() method](https://wicg.github.io/digital-goods/#getDetails-method).

Learn.microsoft.com:
* [StoreProduct.InAppOfferToken Property](/uwp/api/windows.services.store.storeproduct.inappoffertoken).

MDN:
* [Intl.NumberFormat](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) at MDN.
```

Trying to simplify this a bit. I find it harder to read than it needs to be.

In fact, can you globally remove "Draft Community Group Report" in this article? It assumes that developers know what a Community Group is (many don't), and the word "report" is confusing because the document is meant to be a spec (most web developers understand the concept of specs).
I would just use "Digital Goods API specification" throughout.

Reply:
changed from:
in _Digital Goods API: Draft Community Group Report_
to:
in _Digital Goods API specification_

Reply:
removed link above getDetails link